### ISSUE TITLE
Animated custom emoji in admin view should respect autoPlayGif setting

### ISSUE THREAD

**Issue by ThibG** [tootsuite/mastodon#9535]
### Expected behaviour

Custom emoji in `/admin/custom_emojis` should not be animated when the `autoPlayGif` setting is off.
They should be animated on mouse hover instead.

### Actual behaviour

Custom emoji in `/admin/custom_emojis` ignore the `autoPlayGif` setting.

**Comment by nightpool** [comment]
Disagree—it's important that admins know if emoji are animated or not
without having to hover over each one individually.

**Comment by ThibG** [comment]
The original request has been made here: https://sins.center/@Fidgetcetera/101252151281095654

Maybe the animate on hover behavior is not the best one (it would be consistent with what we already have in the interface, though), but I cannot see any reason the setting should not be honored in the admin view as well. Admins may have exactly the same issues with animated content as regular users.

**Comment by Fidgetcetera** [comment]
An "animated" marker on the emoji to indicate that it _is_ animated without showing the actual animation would be great.

**Comment by Fidgetcetera** [comment]
Worst case scenario most hacky solution, just a "show animations" checkbox on the page that defaults based on your settings would help.

**Comment by ThibG** [comment]
Unfortunately, we do not store any information on whether the emoji is animated or not, we systematically generate an animation-free version and use that when `autoPlayGif` isn't set.
So the “show animations” checkbox is probably the easiest option to implement.

**Comment by nightpool** [comment]
here's a concrete, common use-case for this page that I think would be harmed by implementing this:

I get a report from a user that there are a bunch of gross emoji on evil.social. I go to the admin page to look at the evil.social emoji, and I see a bunch of rainbows and flowers. "Wow," I think, "that user is super overreacting". Unbeknownst to me, all of those emoji were two frame animations that switch to guro and cocks (and don't loop!) for anyone who has animated emoji enabled. Because they aren't animated and there's no indication that they're animated, and I don't think to check, my processes and tools mislead my into leaving my users open to this content.

I think an "animated" click-to-play indicator would be a safe way to implement this, but as thibg says, it's a pretty complicated request given the way we currently store deanimated emoji (which arguably we should refactor to make this possible)

**Comment by ThibG** [comment]
@nightpool I have a reverse use-case then: the first frame is offensive, and the second frame isn't, so this messes up with people having `autoPlayGif` and not anyone else.

**Comment by Fidgetcetera** [comment]
if it's that hard to implement we'd definitely opt for the checkbox, and especially in reports (but probably anywhere) making that checkbox have a like, kind of important indicator notifying that emoji might be animated. our issue here is that there are going to be admins who need accessibility features, and if those accessibility features are just ignored on certain screens, that has some really shitty assumptions about who should and should not admin instances. 
it's a great idea to make sure this is implemented in a way that's not possible/easy to abuse, but we just wanna be clear that this kind of accessibility should not be optional because of said abuse, but designed around that instead.
